### ISSUE TITLE
Epic: Create reusable donation components and integrate with Donate, Join Us and About pages

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4492]
#### Issue(s) that need to be completed before work can start on this epic
- [x] #3983 

##### Creating/Integrating New Components
- [x] #4263
- [x] #4544
- [x] #4457
- [x] #4439
- [x] #4504 
- [x] #4507
- [x] #4565

##### Clean Up Issues
- [x] #4454 
- [x] #4455
- [x] #4456
- [x] #4466

### Overview
We need to make donation elements into components that are reusable so that we can replace the ones that are in use across the site

### Details
- The Donate, Join Us and About pages share donation-related elements, however they are not packaged in components suitable for use in future design implementations.
- Also, one element (donation circles graphic) needs to implement a new more accessible design using HTML/CSS. 
- The goal of this epic is to create 3 components and to refactor HTML/CSS so that the appearance of the Donate, Join Us and About pages is modified only in specific ways approved by design and product, and described in [Donation 2023-03-26 deck](https://docs.google.com/presentation/d/1sD5TAtCe7lQkxFMMbNAHU99Z9DZMhwR_vGV5xFC5-q4/edit#slide=id.g20efc25940b_0_9) 

#### Create Issues for Creating New Components
- New Component issues are dependent on completion of 
  - [x]  #3983
- [x] Revise Issue #4263
   - adopt new more accessible Figma design that accommodates larger text and implement in HTML/CSS (without the use of a graphic file)
- [x] New Issue #4457
   - The heading itself will not be part of the component.  
- [x] New Issue #4439 
     - To maximize reusability, the "card: div and the heading "Donate to Hack for LA" will not be part of the component
    -  For mobile screen size, maintain the positioning of the text "*100% of proceeds go to Hack for LA when option is selected" as it appears on the "Donate"page. It should move underneath the animated gif.  
- [x] Create New Issue #4565
  - Both the "Donation Circle" and the "In-Kind" components should be centered within the entire width of the card.

#### Merge Team Instruction
```
#### Merge Team
- [ ] In Epic #4492, check off the dependency under "Creating/Integrating New Issues".  If all of the dependencies under that heading have been merged, then release dependencies on all the issues listed under "Clean Up".
```
#### Create Issues for Incorporating New Components
- [x] New Issue #4504
- [x] New Issue #4507
- [x] New Issue #4544

#### Create Clean up Issues
- Clean up issues are dependent on completion of all the issues under "New Components"
  - [x] New Issue  #4454 
  - [x] New Issue  #4455
  - [x] New Issue  #4456
  - [x] New Issue  #4466 
 
#### Documentation
- [ ] Update notes in [Donation 2023-03-26 deck](https://docs.google.com/presentation/d/1sD5TAtCe7lQkxFMMbNAHU99Z9DZMhwR_vGV5xFC5-q4/edit#slide=id.g20efc25940b_0_9) and summarized below under Goals  with issue numbers. 

### History
- This Epic resulted from #4407
- When the major issues are done, we are going to move to the new donation system #4907

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe Since Bonnie wanted me to review this epic along with the issues in the Dependency section, please add a `ready for dev lead` label and @ mention me in a comment in this epic when you are ready for me to review this epic and its issues. Thanks.

**Comment by roslynwythe** [comment]
- Hi @JessicaLucindaCheng the issues are ready for your review except for #4457 in which I posed a question for Bonnie in a comment.  Thank you for reviewing the Epic and the issues.

**Comment by JessicaLucindaCheng** [comment]
Good job working on writing this complex epic.
- I changed the Overview to be in a modified user format of ` We need to do X for Y reason` and it should be a short summary of the issue. I moved what was originally the longer summary in the Overview section to the Details section and separated the paragraph into bullet points to make it easier to read. If you need more info, you can read [Step 3: Give a good overview in the How to create issues wiki page](https://github.com/hackforla/website/wiki/How-to-create-issues#step-3-give-a-good-overview)
- I added #3983 to the Dependency section and added some subheaders to differentiate between what issues need to be completed before work can start on this epic (and its issues) and what issues are part of the epic itself.

**Comment by roslynwythe** [comment]
> i AM STILL REVIEWING THE EPIC AND ITS ISSUES, SO STILL EDITING THIS COMMENT... Good job working on writing this complex epic.
> 
> * I changed the Overview to be in a modified user format of ` We need to do X for Y reason` and it should be a short summary of the issue. I moved what was originally the longer summary in the Overview section to the Details section and separated the paragraph into bullet points to make it easier to read. If you need more info, you can read [Step 3: Give a good overview in the How to create issues wiki page](https://github.com/hackforla/website/wiki/How-to-create-issues#step-3-give-a-good-overview)
> * I added [Implement Redesigned Donate Page (mobile and desktop) #3983](https://github.com/hackforla/website/issues/3983) to the Dependency section and added some subheaders to differentiate between what issues need to be completed before work can start on this epic (and its issues) and what issues are part of the epic itself.

Thank you @JessicaLucindaCheng.  The meaning of "Dependencies" is so much clearer now.

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe Thank you for your patience while I reviewed this epic and its issues. I appreciate all the hard work you put into writing this complex epic and its issues. :smiley:
- In addition to what I wrote in this comment https://github.com/hackforla/website/issues/4492#issuecomment-1515395323, I commented on each of the issues that you wrote for this epic, except for #4457. 
- When you are ready for me to review #4457, please @ mention me in a comment in the issue and add a `ready for dev lead` label.
- [x] For this epic, please add a `Size` label.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe is checking #4440 to see if it belongs in this epic or is already addressed by another issue.
